.github/workflows/snap.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish Snap
+
+on:
+  # schedule:
+  #   - cron: "0 0 * * *" # Every day at 0:00
+  # workflow_dispatch: # or manually
+  push:
+    branches:
+      - realthunder-snap
```

Comment:
We could use a trigger like "on each push to LinkDaily" for `edge` and "on each release" for `stable` (or `candidate`)

Reply:
Is it possible to trigger build on pushing a tag? I think this way will give more explicit control.

Reply:
Oh yes, sure. Will push later.

Reply:
It would be nice to have different tag formats for `edge` (from LinkDaily) and `stable` (from LinkStage3). Then we can filter for those tags and push to the correct channel.